graphql/handler/server.go

```diff
@@ -25,27 +25,29 @@ type (
 	}
 )
 
+// New returns a new Server for the given executable schema. The Server is not
+// ready for use until the transports you require are added and configured with
+// Server.AddTransport. See the implementation of [NewDefaultServer] for an
+// example.
 func New(es graphql.ExecutableSchema) *Server {
 	return &Server{
 		exec: executor.New(es),
 	}
 }
 
-// NewDefaultServer is a demonstration only. Not for prod.
+// NewDefaultServer returns a Server for the given executable schema which is
+// suitable as an example.
```

Comment:
```suggestion
// suitable **only** as an example of a starting point.
// 
// Please copy this (or something similar) into **your own project** 
// and then customize it to match your specific project needs:
// 
// + Add only the Transports your project needs, in precedence order. 
// + Set up appropriate caches (Memcached? Redis? What size?) 
// + Enable Introspection if required
//
```

Reply:
Using the standard library as an example, I aimed for the following properties:
* The first line of the doc string says what the function does.
* "Deprecated:" preceeds the reason for deprecation and then says what to use instead.
* The documentation concisely states facts and consequences. No emphasis, repetition, or verbosity, which only dilutes the message. The standard library never uses bold type and never says please.

> suitable **only** as an example of a starting point.

The returned Server isn't a starting point, that suggests it can be modified to become production ready, but it can't. The source Code of NewDefaultServer is the starting point, I'll add a commit to make that clearer and include your other recommended configurations.

Reply:
> The returned Server isn't a starting point, my first attempt got this wrong too. It suggests it can be modified to become production ready, but it can't.

Oooooh! Now I see how that could be confusing.